packages/contracts-bedrock/src/libraries/Preinstalls.sol

```diff
@@ -81,12 +87,18 @@ library Preinstalls {
 
+    bytes internal constant SenderCreator_v070Code =
+        hex"6080600436101561000f57600080fd5b6000803560e01c63570e1a361461002557600080fd5b3461018a5760207ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffc36011261018a576004359167ffffffffffffffff9081841161018657366023850112156101865783600401358281116101825736602482870101116101825780601411610182577fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffec810192808411610155577fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe0603f81600b8501160116830190838210908211176101555792846024819482600c60209a968b9960405286845289840196603889018837830101525193013560601c5af1908051911561014d575b5073ffffffffffffffffffffffffffffffffffffffff60405191168152f35b90503861012e565b6024857f4e487b710000000000000000000000000000000000000000000000000000000081526041600452fd5b8380fd5b8280fd5b80fdfea26469706673582212207adef8895ad3393b02fab10a111d85ea80ff35366aa43995f4ea20e67f29200664736f6c63430008170033";
+
+    bytes internal constant EntryPoint_v070Code =
```

Comment:
https://optimistic.etherscan.io/address/0x0000000071727De22E5E9d8BAf0edAc6f37da032

Reply:
Confirmed that this matches

Reply:
Given https://github.com/ethereum-optimism/optimism/pull/10763#issuecomment-2153887360, the sender creator address `0xEFC2c1444eBCC4Db75e7613d20C6a62fF67A167C` should be present in the bytecode of the entrypoint but I cannot find it in there. Curious what you think @dancoombs ?

Reply:
Its in there in 2 places, you have to drop the `0x` and do case-insensitive search.

```
000000000000000000000000efc2c1444ebcc4db75e7613d20c6a62ff67a167c
```

Reply:
Ah good catch, ok thank you!